MetaMask/metamask-extension
# Updated copy change for the IPFS toggle

This PR is to update the copy changes for the IPFS toggle

* Fixes [#1106](https://github.com/MetaMask/MetaMask-planning/issues/1106)

## Screenshots


### Before
![Screenshot 2023-08-02 at 12 43 17 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/67b7bff8-ddbd-468b-8528-50819ebc1db6)


### After

![Screenshot 2023-08-02 at 12 41 48 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/47c25c8d-1553-4853-a77f-283662e813f8)

## Manual Testing Steps


- Search for IPFS in settings
- Check the redirection works properly
- Check the updated copy

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
